.github/workflows/buildtest.yml

```diff
@@ -132,27 +94,49 @@ jobs:
         uses: ./.github/actions/compute-and-check-dependencies-hash
         with:
           docker_image_hash: $DEPENDENCIES_HASH
-      - run: apt update
-      - name: Setup ccache
-        uses: hendrikmuhs/ccache-action@v1.2
-        with:
-          key: ${{ github.job }}-${{ env.CMAKE_PRESET }}
-      - uses: ./.github/actions/configure_4C
-        with:
-          cmake-preset: ${{ env.CMAKE_PRESET }}
-          build-directory: ${{ github.workspace }}/build
-          additional-cmake-flags: -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
       - uses: ./.github/actions/build_4C
         with:
+          cmake-preset: docker_release_clang18
           build-targets: full
           build-directory: ${{ github.workspace }}/build
-      - name: Run minimal tests
+          use-ccache: "true"
+      - uses: ./.github/actions/upload_4C_build
+        with:
+          build-directory: ${{ github.workspace }}/build
+          retention-days: 1
+
+  clang18_test_minimal:
+    needs: clang18_build
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/4c-multiphysics/4c-dependencies:2083744d
+      options: --user root --env OMPI_ALLOW_RUN_AS_ROOT=1 --env OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check docker hash
+        uses: ./.github/actions/compute-and-check-dependencies-hash
+        with:
+          docker_image_hash: $DEPENDENCIES_HASH
+      - name: Setup developer environment for testing
+        run: |
+          cd $GITHUB_WORKSPACE
+          git config --global --add safe.directory $GITHUB_WORKSPACE
+          ./utilities/set_up_dev_env.sh
+      - uses: ./.github/actions/download_4C_build
+        with:
+          build-job: clang18_build
+      - name: Test
         run: |
           cd $GITHUB_WORKSPACE/build
           ctest -L minimal -j `nproc` --output-on-failure
+        env:
+          TEST_CHUNK: ${{ matrix.test-chunk }}
```

Comment:
Why is this here? Does this value even exist?

Reply:
Ah thanks, copy paste error. I will remove it.